exercises/flatten-array/FlattenArrayTest.swift

```diff
@@ -0,0 +1,42 @@
+import XCTest
+
+class FlattenArrayTest: XCTestCase {
+
+    func testFlattenIntegerArray() {
+
+        let result = flattenArray(list: [1, [2, 3, 4, 5, 6, 7], 8])
+        let intResult = result as? [Int]
```

Comment:
This casting would not be nessesary if the functions was a generic constrained to the Element type inside the incoming array.